packages/vats/src/vat-bank.js

```diff
@@ -330,7 +334,10 @@ export function buildRootObject() {
           assert.typeof(issuerName, 'string');
           assert.typeof(proposedName, 'string');
 
-          const brand = await kit.brand;
+          const [brand, displayInfo] = await Promise.all([
+            kit.brand,
+            E(kit.brand).getDisplayInfo(),
```

Comment:
this is awaiting `displayInfo` even if it's not used. is that cheap?

Reply:
no; it's an inter-vat round-trip. good catch. I can postpone this until I'm waiting for the settled issuer.

Reply:
hm... actually, if we needed a round-trip to settle `kit.brand`, the call to `getDisplayInfo()` should get pipelined rather than taking an extra round trip. But `kit.brand` might have been settled in the 1st place, in which case this is an extra round trip.